content/blog/2018/07/2018-07-23-remoting-kafka-plugin-1.0-release.adoc

```diff
@@ -25,7 +25,7 @@ This project aims to develop a plugin in order to add support of a popular messa
 
 === Benefits to the community
 
-* Provide a new method to connect agent to master using Kafka besides existing methods such as JNLP or ssh-slaves-plugin.
+* Provide a new method to connect agent to master using Kafka besides existing methods such as JNLP or ssh-agents-plugin.
```

Comment:
Refer to the plugin by name rather than by the identifier.  The identifier is still `ssh-slaves-plugin` but the name has been changed a while ago to "SSH Build Agents Plugin"
```suggestion
* Provide a new method to connect agent to master using Kafka besides existing methods such as JNLP or plugin:ssh-agents-plugin[ssh Build Agents plugin].
```

Reply:
@MarkEWaite  Did you mean this? 

```suggestion
* Provide a new method to connect agent to master using Kafka besides existing methods such as JNLP or plugin:ssh-slaves-plugin[ssh Build Agents plugin].
```